eosclubhouse/quests/hack2/onboardtour.py

```diff
@@ -0,0 +1,352 @@
+from eosclubhouse.libquest import Quest
+from eosclubhouse.system import Hostname, App
+
+
+class OnboardTour(Quest):
+
+    __tags__ = ['pathway:operating system', 'difficulty:easy']
+    __auto_offer_info__ = {'confirm_before': False, 'start_after': 0}
+    __pathway_order__ = 1
+
+    def step_begin(self):
+        # TODO: Replace with get/set_conf if practical
+        self.comp_isfirst = False
+        self.os_previous = None
+        self.comp_islaptop = Hostname.is_laptop()
+
+        # Apps used
+        self.filesApp = App('org.gnome.Nautilus')
+        self.softwareApp = App('org.gnome.Software')
+        self.settingsApp = App('org.gnome.ControlCenter')
+        self.browserApp = App('org.google.Chrome')
+        # self.officeApp = App('org.libreoffice.LibreOffice.writer')
+        self.officeApp = App('org.libreoffice.LibreOfficeIpc0')
+
+        return self.step_greet
+
+    def step_greet(self):
+        self.onboarding_overview()
+        self.wait_confirm('GREET1')
+        # TODO: wait for 'dialog box moved' signal here
+        self.wait_confirm('GREET2')
+        self.wait_confirm('GREET3')
+        firstcomp_action = self.show_choices_message('Q_FIRSTCOMP',
+                                                     ('NOQUEST_POSITIVE', None, True),
+                                                     ('NOQUEST_NEGATIVE', None, False)).wait()
+        self.comp_isfirst = firstcomp_action.future.result()
+        if self.comp_isfirst:
+            self.wait_confirm('ISFIRSTCOMP')
+            return self.step_teachkbm
+        else:
+            self.wait_confirm('NOTFIRSTCOMP')
+
+        askos_action = self.show_choices_message('Q_PREVOS',
+                                                 ('A_PREVIOUS', None, True),
+                                                 ('A_FRESH', None, False)).wait()
+        if not askos_action.future.result():
+            self.wait_confirm('STARTFRESH')
+        else:
+            self.onboarding_image('temp_mac_win.png', size='50% 16:9')
```

Comment:
I'll discuss about adding temporary assets to master in phabricator. In any case, please remove the "temp" prefix, otherwise we'll have to rename everything when merging the final assets. Let git do the revisioning.